python/sqlcommenter-python/setup.cfg

```diff
@@ -1,5 +1,5 @@
 [flake8]
-exclude = build,.git,.tox
+exclude = build.git.tox
```

Comment:
i thought this is to exclude all `build`, `.tox` and `.git` files. Does the `build.git.tox` work well ?

Reply:
No. Updated again to build,.git,.tox